src/data/seq/seq.lean

```diff
@@ -20,7 +20,11 @@ coinductive seq (α : Type u) : Type u
 A stream `s : option α` is a sequence if `s.nth n = none` implies `s.nth (n + 1) = none`.
 -/
 def stream.is_seq {α : Type u} (s : stream (option α)) : Prop :=
-∀ {n : ℕ}, s n = none → s (n + 1) = none
+∀ (n : ℕ), s n = none → s (n + 1) = none
```

Comment:
I think you probably actually want 
```suggestion
∀ ⦃n : ℕ⦄, s n = none → s (n + 1) = none
```

Reply:
Unfortunately, that seems to cause the most breakage out of the 3 options

Reply:
What breaks? This is the version we use for `function.injective`, for which dot notation works just fine

Reply:
The proofs later on in the file for construction of various `seq`. I can switch to deferred implicit and fix those proofs, if you'd like.